kolibri/plugins/learn/assets/src/modules/pluginModule.js

```diff
@@ -20,7 +21,60 @@ export default {
     };
   },
   actions,
-  getters,
+  getters: {
+    ...getters,
+    learnPageLinks() {
+      const params = {};
```

Comment:
Just a stylistic note, sharing the same and empty `params` object across unrelated routes feels a bit confusing to me.

Reply:
That's a good point @MisRob -- I modeled this after [the pattern shown in the facility plugin](https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/facility/assets/src/modules/pluginModule.js#L46). 

Do you have a recommendation of how to do it differently? 
either 
`params : {}`
or...
not defining it at all unless needed?

Reply:
Ah okay, in the facility plugin it seems that it was needed because there is a shared `facility_id` field in `params` for all routes. In such a case, it'd make sense to me. I didn't see any shared data here and that's why I was wondering why it's been done this way, but it's not a big deal. Up to you :)